morpheushelper/cogs/alert_channel.py

```diff
@@ -0,0 +1,137 @@
+import logging
+from typing import Optional
+
+from PyDrocsid.settings import Settings
+from PyDrocsid.translations import translations
+from discord import TextChannel, Embed, Member, VoiceState
+from discord.ext import commands, tasks
+from discord.ext.commands import Cog, Bot, Context
+
+from colours import Colours
+from permissions import Permission
+from util import send_to_changelog
+
+
+async def get_max_hops() -> int:
+    """
+    Retrieves the channel hops per minute in order for a message to appear
+    """
+    return await Settings.get(int, key="alert_channel_warn_channel_hops", default=5)
```

Comment:
use positional arguments instead of keyword args
```suggestion
    return await Settings.get(int, "alert_channel_warn_channel_hops", 5)
```